scripts/extensions/datetimeField/src/getToggleTemplateDateTimeField.tsx

```diff
@@ -21,7 +23,7 @@ export function getToggleDateTimeField(superdesk: ISuperdesk) {
                             const initialConfig = this.props.config.initial_offset_minutes;
 
                             this.props.setValue(`{{ now|iso_datetime(${initialConfig}) }}`);
-                            notify.success(`Time offset is configured to be ${initialConfig} minutes`);
+                            notify.success(gettext(`Time offset is configured to be ${initialConfig} minutes`));
```

Comment:
in this case we shouldn't use the string literal as that would be looking for a different string for each value, rather it should be a param:
```
notify.success(gettext(`Time offset is configured to be {{x}} minutes`, {x: initialConfig}));
```